spec/controllers/api/internal/helper/purchases_controller_spec.rb

```diff
@@ -54,6 +54,19 @@
         expect(subscription.user).to eq(target_user)
       end
 
+      it "sends grouped receipt to new email and refreshes library" do
```

Comment:
I think I'd make this two tests, these behaviours happen to both be problems for support right now but they're not semantically related